keras_cv/bounding_box/validate.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+
+def validate(bounding_boxes):
+    """validates that a given set of bounding boxes complies with KerasCV format.
+
+    For a set of bounding boxes to be valid it must satisfy the following conditions:
+    - `bounding_boxes` must be a dictionary
```

Comment:
This seems a little suboptimal -- can we always pass around dict and assume it contains "boxes" and "classes"?

Reply:
Can you clarify this comment?  Thats basically what I'm hoping to do here, ensure we always have boxes and classes.  We can change our libraries assumptions, but at the moment all components assume classes to be provided.

Reply:
ok maybe rename it? like `extract_box_info`. Otherwise `validate` sounds like it's not returning anything